app/commands/v2/put_content.rb

```diff
@@ -304,8 +307,13 @@ def raise_if_links_is_provided
         )
       end
 
-      def validate_schema
-        SchemaValidator.new(type: :schema).validate(payload.except(:content_id))
+      def schema_validation_errors
+        @_validation_errors ||=
+          begin
```

Comment:
Ah I see, I thinking it was to swallow exceptions but that makes sense and as there is no `rescure` my idea does not.